COCOA/include/HepMCG4Reader.hh

```diff
@@ -33,23 +33,23 @@
 #define HEPMC_G4_ASCII_READER_H
 
 #include "HepMCG4Interface.hh"
-#include "HepMC/IO_GenEvent.h"
+#include "HepMC3/ReaderRootTree.h"
```

Comment:
Just include ReaderFactory and don't bother about the format.

Reply:
Is there a problem here? I'm not aware of ReaderFactory.

Reply:
Oh, great! Sure, will do.